src/lib.rs

```diff
@@ -775,31 +808,31 @@ impl<M> Recipient<M> {
 
 pub trait SendResultExt {
     /// Don't return an `Err` when the recipient is at full capacity, run `func` in such a case instead.
-    fn on_full<F: FnOnce()>(self, func: F) -> Result<(), DisconnectedError>;
+    fn on_full<F: FnOnce(&'static str)>(self, func: F) -> Result<(), DisconnectedError>;
```

Comment:
Could you document what `&'static str` represents?

Reply:
Good catch, added.